examples/examples-cpu/prefect/prefect-cloud-scheduled-scoring.ipynb

```diff
@@ -103,6 +103,29 @@
     "!prefect auth login -t ${PREFECT_USER_TOKEN}"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<hr>\n",
+    "\n",
+    "## Create a Prefect Cloud Project\n",
+    "\n",
+    "Prefect Cloud organizes flows within workspaces called \"projects\". Before you can register a flow with Prefect Cloud, it's necessary to create a project if you don't have one yet.\n",
+    "\n",
+    "The code below will create a new project in whatever Prefect Cloud tenant you're authenticated with. If that project already exists, this code does not have any side effects."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "client = prefect.Client()\n",
+    "client.create_project(project_name=PREFECT_CLOUD_PROJECT_NAME)"
```

Comment:
will this error if the project already exists?

Reply:
Nope! I tested it multiple times with the same project name.

And here's where `prefect` explicitly wrote code for handling "already exists" as a success case:

https://github.com/PrefectHQ/prefect/blob/81180ad27de7ae3f0e7418bf7f315093654c1847/src/prefect/client/client.py#L981-L991

Reply:
perfect